scalikejdbc-core/src/main/scala/scalikejdbc/TimeUtil.scala

```diff
@@ -0,0 +1,33 @@
+package scalikejdbc
+
+import java.sql.{ Date, Time, Timestamp }
+import java.util.{ Calendar, TimeZone }
+
+/**
+ * Utility functions with respect to time related columns
+ */
+object TimeUtil {
```

Comment:
Newly adding XXXUtil is not preferred (right, we already have LikeConditionEscapeUti though). Just an idea but how about doing like this? Caching `TimeZoneConverter.from(zone1).to(zone2)` would be fine.

```scala
TimeZoneConverter.from(zone1).to(zone2).convert(time)
```